src/Controllers/HomeController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using src.Models;
+
+namespace src.Controllers
+{
+    public class HomeController : Controller
+    {
+        public IActionResult Index()
+        {
+            return View();

```

Comment:
its ok, but we use expression body like this when the method has only one line:
`public IActionResult Index() => View();`